### ISSUE TITLE
The last second of the video is played during "Replay/Close" screen

### ISSUE THREAD

**Issue by SoftVision-CiprianMuresan** [meandavejustice/min-vid#409]
[Affected versions]:
- Firefox 45.0 and up
- Min-Vid 0.3.1-dev

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Latest Min-Vid add-on version (0.3.1-dev) was previously installed on a clean profile.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites. 
2. Open [this](https://www.youtube.com/watch?v=tpr8oqyjKIc&t=37) video in the browser.
3. Turn OFF YouTube autoplay.
4. After the video has ended send it to Min-Vid player.
5. Listen to the video sound as the "Replay/Close" screen is shown.

[Expected result]:
- The video shows the "Replay/Close" screen and nothing else.

[Actual result]:
- The video shows the "Replay/Close" screen, but the last second of the video can be heard.

[Additional Notes]:
- Attached a screen recording of the issue https://goo.gl/6XE8za.

**Comment by SoftVision-PaulOiegas** [comment]
It seams that this is now reproducing for any YouTube video and randomly for Vimeo videos. Any video sent from YouTube is displayed with 1 second less in mini player. That's why the video ends but the soudn can still be heard in the missing second.

![time difference](https://cloud.githubusercontent.com/assets/17258026/22297722/ddbf6326-e326-11e6-9e26-aad0e671d9fb.png)

**Comment by SoftVision-PaulOiegas** [comment]
@meandavejustice It seams that the issue is becoming more severe and even more tangled than we thought.
After reading @BrainStone issue #590, we figured out that the repeat functionality is also affected by this issue. Usually, if the video ends, clicking one on the repeat button doesn't replay it. Instead the missing second is added to the video time. And only if the replay button is clicked the second time, the video replays.

However, there are also videos that do replay in background while the "Repeat" & "Close" overlay is still displayed after first repeat. But this depends on the environment also. We managed to reproduce this scenario on one machine, but on a second it didn't.
